File: Q11.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Q11 {
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) {
        float fahrenheit;
        float resultado;
        System.out.print("Digite uma temperatura em °F: ");
        fahrenheit = leia.nextFloat();
        resultado = (5 * fahrenheit - 160) / 9;
        String celsius = new DecimalFormat("#.0").format(resultado);
        System.out.println("A temperatura em celsius é " + celsius + "°C.");
    }
}

File: Q11.cs
using System;

namespace Q11
{
    class Program
    {
        static void Main(string[] args)
        {
            float fahrenheit, celsius;
            string celsiusf;
            Console.Write("Digite uma temperatura em °F: ");
            fahrenheit = float.Parse(Console.ReadLine());
            celsius = (5 * fahrenheit - 160) / 9;
            celsiusf = string.Format("{0:N1}", celsius);
            Console.WriteLine("A temperatura em celsius é {0}°C.", celsiusf);
        }
    }
}